apollo-compiler/src/test/graphql/com/example/union_inline_fragments/TestQuery.kt

```diff
@@ -410,6 +411,14 @@ class TestQuery : Query<TestQuery.Data, TestQuery.Data, Operation.Variables> {
       }
     }
 
+    override fun toJson(indent: String): String = toJson(indent, DEFAULT)
+
+    override fun toJson(indent: String, scalarTypeAdapters: ScalarTypeAdapters): String {
```

Comment:
I wonder if there is a way to move this implementation to somewhere we can share so that we don't have to duplicate in every class generated.

If we target Java8, then we could use default method implementations.